### PR TITLE
updates for api v3

### PR DESCRIPTION
**Pull request by stefkudla**

### CODE REVIEW

File: src/pages/blog/[slug].astro [hunk 1]
```diff
@@ -43,28 +45,30 @@ const { post } = Astro.props;
 ---
 
 <Layout
-  title={post.title}
+  title={"cats"}
```
Comment (tonyspiro) [inline src/pages/blog/[slug].astro:48]: What's this???

**Review by tonyspiro — APPROVED** [review]
Looks good. Had a questions about why a hard coded value for "cats" was added?

File: src/pages/blog/[slug].astro (hunk 1 above)
Comment (stefkudla) [reply src/pages/blog/[slug].astro]: Honest mistake. Leftover placeholder while debugging

Comment (tonyspiro) [reply src/pages/blog/[slug].astro]: Meow it looks good 🐱

**Pull request merged**